packages/GoogleAnalytics/src/components/WithGATracker/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { FlexComponent } from '@onaio/react-utils';
+import React, { Component } from 'react';
+import { ConnectedComponent } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+import { trackPage } from '../../helpers';
+
+export type ConnectedFlexComponent = ConnectedComponent<FlexComponent<any>, any>;
```

Comment:
@kelvin-muchiri @moshthepitt - Should / could this also go into react-utils?

Reply:
Yes I think we should have it as a peer dependency